android/src/main/java/com/reactnativestripesdk/utils/Mappers.kt

```diff
@@ -346,6 +346,9 @@ internal fun mapFromPaymentMethod(paymentMethod: PaymentMethod): WritableMap {
     it.putString("fingerprint", paymentMethod.card?.fingerprint)
     it.putString("preferredNetwork", paymentMethod.card?.networks?.preferred)
     it.putArray("availableNetworks", paymentMethod.card?.networks?.available?.toList() as? ReadableArray)
+    it.putMap("threeDSecureUsage", WritableNativeMap().also {
+      it.putBoolean("isSupported", paymentMethod.card?.threeDSecureUsage?.isSupported ?: false)
```

Comment:
```suggestion
    it.putMap("threeDSecureUsage", WritableNativeMap().also { threeDSecureUsageMap ->
      threeDSecureUsageMap.putBoolean("isSupported", paymentMethod.card?.threeDSecureUsage?.isSupported ?: false)
```

i think the `it` here will clash with the outer `it`, let's provide an explicit variable name